src/modules/flight_mode_manager/tasks/Utility/StickAccelerationXY.cpp

```diff
@@ -79,7 +79,7 @@ void StickAccelerationXY::generateSetpoints(Vector2f stick_xy, const float yaw,
 		const matrix::Vector2f &vel_sp_feedback, const float dt)
 {
 	// gradually adjust velocity constraint because good tracking is required for the drag estimation
-	if (fabsf(_targeted_velocity_constraint - _current_velocity_constraint) > 0.1f) {
+	if (fabsf(_targeted_velocity_constraint - _current_velocity_constraint) > 0.01f) {
```

Comment:
If I read it correctly this is a guard against infinite values (first condition) and a slew rate for which this class could be used: https://github.com/PX4/PX4-Autopilot/blob/main/src/lib/slew_rate/SlewRate.hpp#L79